src/Files.App/Helpers/AdaptiveLayoutHelpers.cs

```diff
@@ -1,135 +1,137 @@
 using CommunityToolkit.Mvvm.DependencyInjection;
-using Files.Backend.Services.Settings;
+using Files.App.Filesystem;
 using Files.App.ViewModels;
 using Files.App.ViewModels.Previews;
+using Files.Backend.Services.Settings;
+using Files.Shared.Enums;
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using Windows.Storage;
-using System.Collections.Generic;
-using Files.App.Filesystem;
 
 namespace Files.App.Helpers
 {
-    public static class AdaptiveLayoutHelpers
-    {
-        public static bool PredictLayoutMode(FolderSettingsViewModel folderSettings, string path, IList<ListedItem> filesAndFolders)
-        {
-            IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
+	public static class AdaptiveLayoutHelpers
+	{
+		public static bool PredictLayoutMode(FolderSettingsViewModel folderSettings, string path, IList<ListedItem> filesAndFolders)
+		{
+			IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
 
-            if (!userSettingsService.PreferencesSettingsService.ForceLayoutPreferencesOnAllDirectories
-                && folderSettings.IsAdaptiveLayoutEnabled
-                && !folderSettings.IsLayoutModeFixed)
-            {
-                Action layoutDetails = () => folderSettings.ToggleLayoutModeDetailsView(false);
-                Action layoutTiles = () => folderSettings.ToggleLayoutModeTiles(false);
-                Action layoutGridView = () => folderSettings.ToggleLayoutModeGridView(folderSettings.GridViewSize);
+			if (!userSettingsService.PreferencesSettingsService.ForceLayoutPreferencesOnAllDirectories
+				&& folderSettings.IsAdaptiveLayoutEnabled
+				&& !folderSettings.IsLayoutModeFixed
+				&& folderSettings.LayoutMode == FolderLayoutModes.Adaptive)
```

Comment:
Note that this prevents adaptive layout mode from working because folderSettings.LayoutMode is never set to Adaptive.  Even in adaptive mode folderSettings.LayoutMode is still either details or grid or tiles. The above check IsAdaptiveLayoutEnabled is what we want.